ansible/group_vars/all/vars.yml

```diff
@@ -1,9 +1,17 @@
 ---
 git_repo: https://github.com/freeipa/freeipa.git
+ipa_domain: ipa.test
+trusted_domain: trustedipa.test
 ad_root_domain: ad.test
 ad_root_dc_hostname: ad-root
 ad_child_domain: child.ad.test
 ad_child_dc_hostname: ad-child
 ad_tree_domain: adtree.test
 ad_tree_dc_hostname: ad-tree
+ipa_master_hostname: ipamaster
```

Comment:
I'd rather keep the existing names "master", "replica" and "client" instead of "ipamaster", "ipareplica", "ipaclient".